packages/next/src/server/app-render/create-error-handler.tsx

```diff
@@ -84,8 +84,18 @@ export function createErrorHandler({
       }
     }
 
+    // If the error already has a digest, respect the original digest,
+    // so it won't get re-generated into another new error.
+    if (!err.digest) {
+      // TODO-APP: look at using webcrypto instead. Requires a promise to be awaited.
+      const computedDigest = stringHash(
+        err.message + err.stack + (err.digest || '')
```

Comment:
we don't need to consider digest here for the hash since we're no longer hashing it if is already has a digest

Reply:
consider also looking for the component stack on the second arg `errorInfo` which will exist in Dev and during prerenders in prod